Client/WarFare/UIImageTooltipDlg.cpp

```diff
@@ -1065,7 +1065,18 @@ exceptions:;
 			}
 			else
 			{	
-				int iSellPrice = (spItem->pItemBasic->iPrice*spItem->pItemExt->siPriceMultiply/6);
+				int iSellPrice = 0;
+
+				if (spItem->pItemBasic->iSaleType == 1) 
+				{   //sale type = 1, selling for the same price of buying
```

Comment:
With an enum making this clear, this comment isn't necessary, but your comments should always have a leading space, i.e.:
`// comment`
(it also shouldn't be on {)